huaweicloud/provider.go

```diff
@@ -58,7 +58,7 @@ func Provider() *schema.Provider {
 		Schema: map[string]*schema.Schema{
 			"region": {
 				Type:         schema.TypeString,
-				Required:     true,
+				Optional:     true,
```

Comment:
if the region is only optional under profile authentication, we add a check to avoid unexcept error

Reply:
done